pkg/lclient/fakeclient.go

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
Please confirm with legal that we can have this copyright here. I don't think it'd be compatible with our current license.

Reply:
That was added erroneously and should not be there, removed.